drink/src/session.rs

```diff
@@ -184,18 +179,8 @@ impl<R: Runtime> Session<R> {
         self.transcoders.register(contract_address, transcoder);
     }
 
-    /// Returns a reference for basic chain API.
-    pub fn chain_api(&mut self) -> &mut impl ChainApi<R> {
-        &mut self.sandbox
-    }
-
-    /// Returns a reference for basic contracts API.
-    pub fn contracts_api(&mut self) -> &mut impl ContractApi<R> {
-        &mut self.sandbox
-    }
-
-    /// Returns a reference for mocking API.
-    pub fn mocking_api(&mut self) -> &mut impl MockingApi<R> {
+    /// The underlying `Sandbox` instance.
+    pub fn sandbox(&mut self) -> &mut Sandbox<R> {
```

Comment:
what is your reasoning for this? I have no strong opinion on that, but my initial intention was to express precisely what functionalities you are going to use

Reply:
oh, now I see that you are removing api traits at all

Reply:
commented below. We can add these traits back if we implement them with the right constraints, but I don't think this add much value.